### DISCUSSION TITLE
Static content compilation error: Operation on an invalid type on /vendor/swissup/theme-frontend-breeze-blank/web/css/default.less

### DISCUSSION THREAD

**Discussion by acromatica** [breezefront/community#338] [1. Help]
Hi!

I'm working on a custom theme that is based on Breeze. I've followed the steps from the guide and created a couple of css files. When I'm trying to compile it with `setup:static-content:deploy` I'm getting the following error:

`Compilation from source: /Users/username/Projects/projectname/vendor/swissup/theme-frontend-breeze-blank/web/css/default.less
Operation on an invalid type`

`Error happened during deploy process: Magento\Framework\Exception\LocalizedException::__construct(): Argument #1 ($phrase) must be of type Magento\Framework\Phrase, string given, called in /Users/username/Projects/projectname/vendor/magento/module-deploy/Service/DeployPackage.php on line 142`

**Comment by vovayatsyuk** [comment]
What changes have you made? 

`Operation on an invalid type` error may happen when a LESS variable is changed to the unsupported format.

**Reply by acromatica** [reply]
It seemed to be due to unescaped calc() functions in my less files. Once I escaped them it started compiling ¯\_(ツ)_/¯
